tests/plotting/test_plotting.py

```diff
@@ -1224,10 +1238,10 @@ def test_scalars_by_name():
     plotter = pyvista.Plotter()
     data = examples.load_uniform()
     plotter.add_mesh(data, scalars='Spatial Cell Data')
-    plotter.show(before_close_callback=verify_cache_image)
+    plotter.show()
 
 
-def test_multi_block_plot():
+def test_multi_block_plot(request):
```

Comment:
The fixture doesn't seem to be used in the test.
```suggestion
def test_multi_block_plot():
```